xbmc/addons/kodi-dev-kit/include/kodi/addon-instance/PVR.h

```diff
@@ -3298,26 +3287,28 @@ class ATTR_DLL_LOCAL CInstancePVRClient : public IAddonInstance
   //--==----==----==----==----==----==----==----==----==----==----==----==----==
 
   inline static PVR_ERROR ADDON_GetTimerTypes(const AddonInstance_PVR* instance,
-                                              PVR_TIMER_TYPE* types,
-                                              int* typesCount)
+                                              PVR_TIMER_TYPE*** types,
+                                              unsigned int* typesCount)
```

Comment:
Can be also leaved as is and OK for me with `unsigned int` 👍. 
Only thought about relationships to use `size_t` here and other places as relates to list arrays (background in my mind was to identify in autogen scripts his relation to value before).

Reply:
I want to keep the changes in this PR down to a minimum. Changing the type to `size_t` here and at all other places in the C API (there are plenty of them)  most probably introduces another bunch of compiler warnings for type conversions which I would like to avoid.